src/react-components/room/ChatSidebarContainer.js

```diff
@@ -164,12 +180,15 @@ export function ChatSidebarContainer({ scene, canSpawnMessages, presences, occup
   const { messageGroups, sendMessage, setMessagesRead } = useContext(ChatContext);
   const [onScrollList, listRef, scrolledToBottom] = useMaintainScrollPosition(messageGroups);
   const [message, setMessage] = useState("");
+  const { text_chat: canTextChat, kick_users: isMod } = usePermissions();
+  const { text_chat: textChatEnabled } = useRoomPermissions();
   const typingTimeoutRef = useRef();
   const intl = useIntl();
   const inputRef = useRef();
 
   const onKeyDown = useCallback(
     e => {
+      if (!canTextChat) return;
```

Comment:
To allow slash commands I think we can basically just check if the value starts with a `/` here.